gcp-cloud-run/02-provider.tf

```diff
@@ -1,10 +0,0 @@
-
-# provider "google" {
```

Comment:
Some customers uses this module as a project directly. In that case we instruct them to uncomment those lines and `terraform apply`. This is the reason we kept this file here.

Reply:
Ok, lets keep it for now and look into publishing the module into the Terraform registry later.